tests/codes/eurocode/nen_en_1993_1_9_c2_2012/annex_a_determination_of_fatigue_load_parameters_and_verification_formats/test_formula_a_2.py

```diff
@@ -0,0 +1,65 @@
+"""Testing formula A.1 from NEN-EN 1993-1-9+C2:2012: Annex A - Determination of fatigue load parameters and verification formats."""
+
+import pytest
+
+from blueprints.codes.eurocode.nen_en_1993_1_9_c2_2012.annex_a_determination_of_fatigue_load_parameters_and_verification_formats.formula_a_2 import (
+    FormADot2CriteriaBasedOnDamageAccumulation,
```

Comment:
Import is sorted in a way you can do multiple function imports at once. in this case this is not neccesary. The "(" "," and ")" can be removed.

Reply:
Is forced by Black extention. Proposal to keep format.